### DISCUSSION TITLE
Disabling expandOnClick doesn't work

### DISCUSSION THREAD

**Discussion by RinaZabag** [apexcharts/apexcharts.js#3591] [Q&A]
Hi,
I'm using apexchart.js 3.36.3 and vue-apexcharts 1.6.2 and I'm trying to disable the expand on click on my pie chart but it not being disabled. Here is my configuration of the char options:
```
plotOptions: {
  pie: {
    expandOnClick: false,
    donut: {
      labels: {
        show: true,
      },
    },
  },
},
 ```
Are there additional properties I need to add in order this to work?

**Comment by renanloboz** [comment]
Hey  @RinaZabag,
I know it’s been a while, but I found a solution for this in React:

```javascript
options: {
  // ...,
  states: {
    active: {
      filter: {
        type: 'none',
      }
    }
  }
}
```
Hope it helps :)
